src/sidebar/test/cross-origin-rpc-test.js

```diff
@@ -70,12 +71,13 @@ describe('sidebar/cross-origin-rpc', () => {
       );
     });
 
-    describe('changeFocusModeUser', () => {
-      function callRPC(params) {
+    describe('on RPC request from embedder frame', () => {
```

Comment:
The `changeFocusModeUser` tests haven't changed here, they have just been wrapped in a new describe block.